programs/mango-v4/src/state/mango_account.rs

```diff
@@ -88,7 +88,9 @@ pub struct MangoAccount {
 
     pub frozen_until: u64,
 
-    pub reserved: [u8; 232],
+    pub fees_accrued: u64,
```

Comment:
I don't like the name, since it's not tracking the overall fees_accrued at all. Instead it tracks only some fees. And it gets decreased when the MNGO->USDC conversion gets used.